Userland/Utilities/tail.cpp

```diff
@@ -87,18 +85,15 @@ int main(int argc, char* argv[])
     args_parser.add_option(follow, "Output data as it is written to the file", "follow", 'f');
     args_parser.add_option(line_count, "Fetch the specified number of lines", "lines", 'n', "number");
     args_parser.add_positional_argument(file, "File path", "file");
-    args_parser.parse(argc, argv);
+    args_parser.parse(args);
 
     auto f = Core::File::construct(file);
     if (!f->open(Core::OpenMode::ReadOnly)) {
         warnln("Failed to open {}: {}", f->name(), f->error_string());
         exit(1);
     }
```

Comment:
Instead of using `Core::File::construct()` you can use the TRYable `Core::File::open()`